src/ripple/test/impl/JSONRPCClient.cpp

```diff
@@ -116,6 +116,11 @@ class JSONRPCClient : public AbstractClient
         {
             Json::Value jr;
             jr["method"] = cmd;
+#if 1  // force use of 2.0 API
+            jr["jsonrpc"] = "2.0";
+            jr["ripplerpc"] = "2.0";
+            jr["id"] = 5;
```

Comment:
Working on this switch now, and will add the appropriate additions to the test for V2.